tests/test-document-fuse.sh

```diff
@@ -49,20 +49,24 @@ export XDG_RUNTIME_DIR=${TEST_DATA_DIR}/runtime
 cleanup () {
     fusermount -u $XDG_RUNTIME_DIR/doc || :
     sleep 0.1
-    /bin/kill -9 $DBUS_SESSION_BUS_PID
+    kill -9 $DBUS_SESSION_BUS_PID
     kill $(jobs -p) &> /dev/null || true
     rm -rf $TEST_DATA_DIR
 }
 trap cleanup EXIT
 
-sed s#@testdir@#${test_builddir}# ${test_srcdir}/session.conf.in > session.conf
+conf_template="${test_srcdir}/session.conf.in"
+if [[ ! -f "$conf_template" ]]; then
+    conf_template="${test_srcdir}/tests/session.conf.in"
+fi
+sed s#@testdir@#${test_builddir}# $conf_template > session.conf
```

Comment:
This is quite ugly. It would be better if we could make install to the same directory but I do not understand why

https://github.com/flatpak/xdg-desktop-portal/blob/b907d57fd6e45bebb4aca71f50b7017dbea7fc5a/tests/Makefile.am.inc#L112-L113

places it into `tests` subdirectory, while

https://github.com/flatpak/xdg-desktop-portal/blob/b907d57fd6e45bebb4aca71f50b7017dbea7fc5a/tests/Makefile.am.inc#L116-L117

does not.

Reply:
Reading `glib-tap.mk`:

> Note that _data is always used with the nobase_ automake variable name to ensure that installed test data is installed in the same way as it appears in the package layout.

Is adding it to `installed_test_DATA` and `EXTRA_DIST` instead an acceptable solution?

Reply:
I think this is because `glib-tap.mk` was designed with recursive `make` in mind, but xdg-desktop-portal is using non-recursive `make`, so the expectations are different. Further complicating things, the "upstream" version of `glib-tap.mk` was removed from GLib 2.59 with the removal of Autotools, and the version in xdg-desktop-portal has been patched anyway.

> Is adding it to `installed_test_DATA` and `EXTRA_DIST` instead an acceptable solution?

I would personally put it in `dist_installed_test_DATA`, with the capitals, instead (which is more or less equivalent to the combination of `installed_test_DATA` and `EXTRA_DIST`, but without the duplication).